ci: Add note about file_diff_from_base for GHA

seemethere opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#60110 ci: Add note about file_diff_from_base for GHA**

file_diff_from_base is currently bugged for ghstack PRs since it fails
to find a merge base

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

Differential Revision: [D29168767](https://our.internmc.facebook.com/intern/diff/D29168767)

seemethere committed:
ci: Add note about file_diff_from_base for GHA

file_diff_from_base is currently bugged for ghstack PRs since it fails
to find a merge base

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

[ghstack-poisoned]

seemethere commented:
@seemethere has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29168767).

driazati commented on .jenkins/pytorch/macos-test.sh:
```diff
@@ -53,7 +53,7 @@ test_python_all() {
   # Try to pull value from CIRCLE_PULL_REQUEST first then GITHUB_HEAD_REF second
   # CIRCLE_PULL_REQUEST comes from CircleCI
-  # GITHUB_HEAD_REF comes from Github Actions
+  # NOTE: file_diff_from_base is currently bugged for GHA due to an issue finding a merge base for ghstack PRs
```
can these link to a GitHub issue for more context/tracking?

seemethere replied:
Added a reference to https://github.com/pytorch/pytorch/issues/60111

seemethere committed:
Update on "ci: Add note about file_diff_from_base for GHA"


file_diff_from_base is currently bugged for ghstack PRs since it fails
to find a merge base

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

Differential Revision: [D29168767](https://our.internmc.facebook.com/intern/diff/D29168767)

[ghstack-poisoned]

seemethere commented:
@seemethere has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29168767).
